packages/types/scripts/generate.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable */
+// @ts-nocheck
```

Comment:
I don't see the point of having TypeScript and eslint if we suppress everything anyway...

Reply:
Updated 87ea767 , removed almost all eslint disable and no ckeck rules in not autogenerated files.